core/src/util/inputHandler.cpp

```diff
@@ -0,0 +1,124 @@
+#include "inputHandler.h"
+
+#include <cmath>
+#include "glm/glm.hpp"
+#include "platform.h"
+
+InputHandler::InputHandler(std::shared_ptr<View> _view) : m_view(_view) {
+    m_deltaZoom = 0.f;
+    m_deltaTranslate = glm::vec2(0.f);
+}
+
+void InputHandler::update(float _dt) {
+
+    bool hasMomentum = glm::length(m_deltaTranslate) > m_minDeltaLength || std::fabs(m_deltaZoom) > m_minDeltaZoomLength;
```

Comment:
I see you are using std::abs and std::fabs both in the code.
c++ has std::abs overloaded to take int/float/double values. It should suffice to use std::abs here (also for consistency).